fem/gridfunc.cpp

```diff
@@ -2359,6 +2359,9 @@ void GridFunction::ProjectDeltaCoefficient(DeltaCoefficient &delta_coeff,
 
 void GridFunction::ProjectCoefficient(Coefficient &coeff)
 {
+   MFEM_VERIFY(
+      fes->GetVDim() == 1,
```

Comment:
We might want to use `GridFunction::VectorDim` instead of `fes->GetVDim()` (basically the same, but seems more direct to me).

```suggestion
      VectorDim() == 1,
```

Reply:
Good suggestion! I was trying to figure out how to get this to work for VectorCoefficients and I think this works for that too.

Reply:
They are not the same, `VectorDim()` takes into account the range dimension, which is good to test here anyway 😉 .